schemas/questionnaire_v1.json

```diff
@@ -71,6 +71,31 @@
       "pattern": "^GB-[A-Z]{3}$",
       "description": "An ISO 3166-2:GB region code"
     },
+    "answer_codes": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "items": {
+        "type": "object",
+        "properties": {
+          "code": {
+            "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
+            "pattern": "^[a-zA-Z0-9-_]+$",
```

Comment:
This should match the regex in the JSON schema definition on ons-schema-definitions. I think `dots` seperations missing.

Reply:
Updated now 👍